### PR TITLE
Created kubevirt overview page which combines vms and vmis into one view

### PR DESCRIPTION
**Pull request by kevinylee1991**
Overview page:
Screen Shot 2022-06-08 at 5 40 01 PM

The migration status message:
Screen Shot 2022-07-11 at 6 43 42 PM

VM details page:
Screen Shot 2022-07-12 at 9 58 41 AM

### CODE REVIEW

File: src/app/plugins/kubevirt/components/virtual-machines/vm-details/Overview.tsx [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+      }
+      return fields
+    }
+  }, [vm])
+
+  const memoryData = useMemo(() => {
+    return getFieldsForCard(memoryFields, vm)
+  }, [memoryFields])
+
+  const disks = useMemo(() => {
```
Comment (jruddell) [inline src/app/plugins/kubevirt/components/virtual-machines/vm-details/Overview.tsx:250]: hmm maybe this should be inside a selector?

**Review by jruddell — APPROVED** [review]

File: src/app/plugins/kubevirt/components/virtual-machines/vm-details/Overview.tsx (hunk 1 above)
Comment (kevinylee1991) [reply src/app/plugins/kubevirt/components/virtual-machines/vm-details/Overview.tsx]: I can move CPU and memory in, something similar to the networks

File: src/app/plugins/kubernetes/components/common/entity/entity-events-page.tsx [hunk 2]
```diff
@@ -85,6 +85,7 @@ export default function EntityEventsPage({ entity, loading = false }) {
     requiredParams,
   })
   const allEvents = useSelectorWithParams(clusterEventsSelector, params)
+  console.log(allEvents, 'allEvents')
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/common/entity/entity-events-page.tsx:88]: console.log

File: src/app/plugins/kubevirt/components/virtual-machines/selectors.ts [hunk 3]
```diff
@@ -74,25 +144,87 @@ export const virtualMachinesSelector = createSharedSelector(
...
+      return accum
+    }, {})
+    const vmisWithNoVm = vmis.filter((vmi) => {
+      return !vmisFromVmsMap[vmi.id]
+    })
+    return [...combinedFromVms(vms), ...combinedFromVmis(vmisWithNoVm)]
+  },
+)
+
+export const makeAllVmsSelector = (
```
Comment (jruddell) [inline src/app/plugins/kubevirt/components/virtual-machines/selectors.ts:215]: Is this used? I don't think we need make selectors anymore

**Review by jruddell — APPROVED** [review]

**Pull request merged**